### PR TITLE
assignment-1

### PR DESCRIPTION
**Pull request by YuriyKoshulap**
TITLE: UofT-DSI | shell - Assignment-1

## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
adding code with comments
## What did you learn from the changes you have made?
there are multiple ways of achieving the same task; how to use "--help" in bash to get detailed info on bash commands
## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
concerning bash, some answers include several alternatives; in terms of GitHub - ultimately I'd like to commit via VS Code entirely

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
I'm a Linux fan, so (this level of) Bash is easy, not familiar with GitHub, 

## How were these changes tested?
GitBash

## @mentions of the person or team responsible for reviewing proposed changes (At least 2 people)
@michaeladrouillard
@dtxe
- 

## Checklist
[x] I can confirm that my changes are working as intended
`uoft-dsi-c3-homework-20240425`

### CODE REVIEW

**Comment by YuriyKoshulap** [comment]
@michaeladrouillard
@dtxe

**Comment by YuriyKoshulap** [comment]
@michaeladrouillard
@dtxe

File: 03_homework/homework.sh [hunk 1]
```diff
@@ -7,21 +7,49 @@
 # On your terminal, input all the commands you have used to create the following:
 
 # 1. How would you create 5 directories? Feel free to use any name for your directories.
+mkdir dir1 dir2 dir3 dir4 dir5
 
 # 2. How would you verify the creation of all 5 directories?
+ls -a
 
 # 3. In each directory, how would you create 5 .txt files and write "I love data" into each within the directories?
+touch dir1/file1.txt dir2/file2.txt dir3/file3.txt dir4/file4.txt dir5/file5.txt
```
Comment (dwz92) [inline 03_homework/homework.sh:16]: Need write `I love data` into these files

**Review by dwz92 — APPROVED** [review]

File: 03_homework/homework.sh (hunk 1 above)
Comment (YuriyKoshulap) [reply 03_homework/homework.sh:16]: Missed that, thank you, Emma!

**Review by YuriyKoshulap — COMMENTED** [review]
Solution updated as prompted by the instructor.

**Comment by YuriyKoshulap** [comment]
`uoft-dsi-c3-homework-20240425`

**Review by juliagallucci — APPROVED** [review]
Great!

**Pull request closed without merging**